yarn.lock

```diff
@@ -0,0 +1,3114 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I could make this an npm lockfile if you prefer.

The advantage of having this is that in the event that a dependency of envalid is broken you can see from this lockfile what exact versions of modules are working, and then debug from this point.